framework_tvb/tvb/adapters/analyzers/bct_adapters.py

```diff
@@ -169,25 +175,26 @@ def get_form_class(self):
         return BaseUnidirectedBCTForm
 
     @abstractmethod
-    def launch(self, connectivity, **kwargs):
+    def launch(self, view_model):
         pass
 
-
 class ModularityOCSM(BaseBCT):
     """
     """
     _ui_group = BCT_GROUP_MODULARITY
     _ui_connectivity_label = "Directed (weighted or binary) connection matrix:"
```

Comment:
Field `_ui_connectivity_label` should no longer be used, right?
If so, it can be removed